network/src/commonMain/kotlin/com/wire/kalium/network/api/conversation/ConversationEvent.kt

```diff
@@ -1,65 +1,23 @@
 package com.wire.kalium.network.api.conversation
 
-import com.wire.kalium.network.api.ConversationId
 import com.wire.kalium.network.api.UserId
-import com.wire.kalium.network.api.model.ConversationAccess
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class ConversationEvent(
-    @SerialName("type") val eventType: String,
-    @SerialName("data") val data: EventData,
-    @Deprecated("use qualified id")
-    @SerialName("conversation") val conversationId: String?,
-    @SerialName("qualified_conversation") val qualifiedConversationId: ConversationId,
-    @Deprecated("use qualified from")
-    @SerialName("from") val from: String?,
-    @SerialName("qualified_from") val fromUser: UserId,
-    @SerialName("time") val time: String
+data class ConversationMember(
+    @SerialName("conversation_role") val conversationRole: String?, // Role name, between 2 and 128 chars, 'wire_' prefix is reserved for roles designed by Wire (i.e., no custom roles can have the same prefix)
+    @SerialName("qualified_id") val qualifiedId: UserId
 )
 
 @Serializable
-data class EventData(
```

Comment:
This class is used in `AddParticipantResponse`

Reply:
Yes, but I talked to Vitor about it. He said we need to get rid of it, also the field in the AddParticipantResponse is not used anywhere.